src/client.ts

```diff
@@ -231,10 +235,14 @@ export class Client {
 
   private _sendToHost<T, V>(partialMsg: PartialMsg<T, V>): void {
     const message = applyProtocol(partialMsg);
-    let validated = null;
+
+    let validated: ClientToHost;
 
     try {
       validated = validateOutgoing(message);
+      if (validated.direction === undefined) {
+        validated.direction = 'ClientToHost';
+      }
```

Comment:
Same comment as on the host side: I don't think we need/want the conditional. Everything outbound from here should always be flagged as `ClientToHost`